src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -19,14 +19,8 @@
  */
 package ch.njol.skript.variables;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;

```

Comment:
Don't do * imports. Skript does not use them and we want some consistency.

Reply:
Blame IntelliJ on that. I'll change it.